Assets/Editor/TestResponseDeserialization.cs

```diff
@@ -73,7 +73,7 @@ public class TestResponseDeserialization {
 
             Assert.IsNull(response.DataJSON);
             Assert.IsNotNull(response.errors);
-            Assert.AreEqual("Field 'doesntExist' doesn't exist on type 'Shop'", response.errors[0]);
+            Assert.AreEqual("[\"Field 'doesntExist' doesn't exist on type 'Shop'\"]", response.errors);
```

Comment:
Right now `ParseErrors` from `TopLevelResponse` only pulls out the message of the error - not the entire dictionary. Do we want to modify that to take return back the entire GraphQL error dictionary?